swan-lake/development-tutorials/test-document-the-code/ballerina-profiler.md

```diff
@@ -0,0 +1,365 @@
+---
+title: Ballerina Profiler
+description: Check out how you can run and do a CPU profile of a Ballerina program.
```

Comment:
```suggestion
description: Check out how you can run and do a CPU profile of a Ballerina program.
```
What is meant by "doing" a CPU profile?

Reply:
Will remove that and just say profile a Ballerina program.